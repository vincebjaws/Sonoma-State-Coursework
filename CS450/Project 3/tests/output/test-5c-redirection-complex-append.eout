twu
479826 /usr/share/dict/words
