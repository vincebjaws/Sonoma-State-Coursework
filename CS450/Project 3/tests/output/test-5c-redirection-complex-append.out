 479826 /usr/share/dict/words
      1 tests/output/tmp
 479827 total
twu
